win32ss/reactx/ntddraw/d3d.c

```diff
@@ -64,11 +64,11 @@ NtGdiDdCanCreateD3DBuffer(HANDLE hDirectDraw,
 
     if (pfnDdCanCreateD3DBuffer == NULL)
     {
-        DPRINT1("Warring no pfnDdCanCreateD3DBuffer");
+        DPRINT1("Warning: no pfnDdCanCreateD3DBuffer\n");
         return DDHAL_DRIVER_NOTHANDLED;
     }
 
-    DPRINT1("Calling on dxg.sys DdCanCreateD3DBuffer");
+    DPRINT("Calling on dxg.sys pfnDdCanCreateD3DBuffer\n");
```

Comment:
Probably "Calling on dxg.sys" could be changed to just "Calling dxg.sys" on this and the next 14 cases, but not a big deal. I will approve anyway. Thanks.